### PR TITLE
add-cart-item

### PR DESCRIPTION
**Pull request by D-Kondratiev**
##Description added cart item 
screenshot link: 
https://snipboard.io/D56gtU.jpg

### CODE REVIEW

**Review by kinqbert — APPROVED** [review]
Great job! Approved by me!

File: src/components/CartItem/CartItem.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
+@import '../../prepare-styles/mixins.scss';
+@import '../../prepare-styles/variables.scss';
+@import '../../prepare-styles/colors.scss';
+@import '../../prepare-styles/usable-fonts.scss';
+
+
+
+.card-item__container {
+  font-family: 'Mont-SemiBold', sans-serif;
+  border: 1px solid #e2e6e9;
```
Comment (ilyasolo881) [inline src/components/CartItem/CartItem.scss:10]: use color variable

File: src/components/CartItem/CartItem.scss [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  height: 160px;
+  margin-bottom: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fff;
```
Comment (ilyasolo881) [inline src/components/CartItem/CartItem.scss:16]: use color variable

File: src/components/CartItem/CartItem.scss [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  grid-area: 2 / 1 / 3 / 3;
+}
+
+.cart-item__buttons-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 32px;
+  height: 32px;
+  background-color: #fff;
```
Comment (ilyasolo881) [inline src/components/CartItem/CartItem.scss:95]: use color variable

**Pull request merged**